### ISSUE TITLE
Always show deposit/withdraw buttons for vaults

### ISSUE THREAD

**Issue by dudesahn** [iearn-finance/yearn-finance#94]
Comment from x48: 

- If wallet is not connected button should show as disabled
 - opacity: .5
 - cursor: not-allowed
- If wallet is not connected show tool tip with copy (“Connect your wallet to deposit (or withdraw)”) (blocked --need tooltip copy from copy team… unblock yourself by finding the copy team… ask x48)

**Comment by Faolain** [comment]
Closed #99 and reopened a new PR at https://github.com/iearn-finance/yearn-finance/pull/138

**Comment by Faolain** [comment]
This can be moved to done I believe as it's been merged
